pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -76,6 +76,8 @@ var (
 	defaultDefaultProject, defaultMetadataProject, defaultDNSProject, defaultDefaultServiceAccount string
 	// projects for which a cron GC job exists.
 	projectsWithGC []string
+	// defaultUseBulkInsert controls whether to use the SDK-based BulkInsert API for creating VMs.
+	defaultUseBulkInsert bool
```

Comment:
nit: I don't think we need a "default" for this, considering it's just a boolean. What about implementing it like how managed instances (`providerOpts.Managed`) is, i.e. as a `ProviderOpts` field that's hook up to CLI flags?